conf/logback.xml

```diff
@@ -15,6 +15,15 @@
         </encoder>
     </appender>
 
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%date [%level] from %logger in %thread - %message%n%xException</pattern>
```

Comment:
I don't know what the right format is off the top of my head, but I assume this will work for our purposes

Reply:
In the longer-term I think we will be better off using structured/JSON log lines, but this should be OK for the proof of concept!